### DISCUSSION TITLE
Dynamic addressing

### DISCUSSION THREAD

**Discussion by malaki86** [gioblu/PJON#410] [Q&A]
I'm working on an ESP 32 project that will use both ESPNow and SoftwareItBang strategies.   I was looking at the dynamic addressing and it seems vital that I include this, but I can't find any samples or examples of actually using it. Does anyone have any that they can share? It will be a single Master with multiple slaves.

**Comment by gioblu** [comment]
Ciao @malaki86 I have answered you already here: https://github.com/gioblu/PJON/issues/411
Feel free to write here if you need support.
